src/Fixer/FunctionNotation/MethodArgumentSpaceFixer.php

```diff
@@ -97,6 +102,48 @@ private function fixFunction(Tokens $tokens, $startFunctionIndex)
             if ($token->equals(',')) {
                 $this->fixSpace2($tokens, $index);
             }
+
+            if ($this->isNewLine($token)) {
+                $isMultiline = true;
+            }
+        }
+
+        return $isMultiline;
+    }
+
+    private function ensureFunctionFullyMultiline(Tokens $tokens, $startFunctionIndex)
```

Comment:
yes, no worries, follow the interfaces and name things a bit logical and all good :)